File: chat-uikit/src/main/java/io/agora/chat/uikit/widget/photoview/SDK16.java
package io.agora.chat.uikit.widget.photoview;

import android.annotation.TargetApi;
import android.view.View;

@TargetApi(16)
class SDK16 {

	public static void postOnAnimation(View view, Runnable r) {
		view.postOnAnimation(r);
	}
	
}

File: ease-im-kit/src/main/kotlin/io/agora/uikit/widget/photoview/SDK16.kt
package io.agora.uikit.widget.photoview

import android.annotation.TargetApi
import android.view.View

@TargetApi(16)
internal object SDK16 {
    fun postOnAnimation(view: View, r: Runnable?) {
        view.postOnAnimation(r)
    }
}